hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/LocalRegionFactoryDefaultTest.java

```diff
@@ -60,7 +61,95 @@ protected Properties getCacheProperties() {
     }
 
     @Test
-    public void testEntity() {
+    public void testNonStrictReadWriteEntity() {
```

Comment:
A good example of a test doing to much.You have 2 groups of asserts; this means you have 2 tests (potentially more) in a single method. A test method should be short in most cases (10/15 lines).